bot/seasons/easter/egg_facts.py

```diff
@@ -0,0 +1,64 @@
+import asyncio
+import logging
+import random
+from json import load
+from pathlib import Path
+
+import discord
+from discord.ext import commands
+
+from bot.constants import Channels
+from bot.constants import Colours
+
+
+log = logging.getLogger(__name__)
+
+
+class EasterFacts(commands.Cog):
+    """
+    A cog contains a command that will return an easter egg fact when called.
+
+    It also contains a background task which sends an easter egg fact in the event channel everyday.
+    """
+
+    def __init__(self, bot):
+        self.bot = bot
+        self.facts = self.load_json()
+
+    @staticmethod
+    def load_json():
+        """Load a list of easter egg facts from the resource JSON file."""
+        p = Path('bot', 'resources', 'easter', 'easter_egg_facts.json')
+        with p.open(encoding="utf8") as f:
+            facts = load(f)
+        return facts
+
+    async def send_egg_fact_daily(self):
+        """A background task that sends an easter egg fact in the event channel everyday."""
+        channel = Channels.seasonalbot_chat
```

Comment:
You still need `self.bot.get_channel()` here to get the `Channel` object, `Channels.seasonalbot_chat` is just an integer.

You're going to need to merge in the upstream changes to `master` before this will work. Your fork is behind where this constant was added to `constants.py`.

Reply:
ok